### ISSUE TITLE
[GLPI 10 Bugfixes] Waiting reason & technician followup

### ISSUE THREAD

**Issue by tsmr** [glpi-project/glpi#13635]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

[GLPI 10 Bugfixes]

### Bug description

When a ticket is pending with a pending reason, if a technician adding a follow-up,  the ticket stays in pending status but the pending reason is removed

So if we want to put it back, we have to go back to the ticket in progress, then put it back on waiting with the original reason

**Comment by leonardosaur** [comment]
The automatic follow-up from cron also remove the pending reason and set the ticket status to "Atributted", inviabilizing the use of the "Pending reason". Ex.: sometimes i want to alert 3 or 4 times and each follow-up the pending reason is removed and status is set to atributted.

**Comment by AdrienClairembault** [comment]
Yes (there has been a couple PR regarding this so not sure which exact one fix this particular issue tho).
